mne/dipole.py

```diff
@@ -41,15 +42,15 @@ class Dipole(object):
     Parameters
     ----------
     times : array, shape (n_dipoles,)
-        The time instants at which each dipole was fitted.
+        The time instants at which each dipole was fitted (sec).
     pos : array, shape (n_dipoles, 3)
         The dipoles positions in meters
     amplitude : array, shape (n_dipoles,)
-        The amplitude of the dipoles in nAm
+        The amplitude of the dipoles (nA).
     ori : array, shape (n_dipoles, 3)
-        The dipolar moments. Amplitude of the moment is in nAm.
+        The dipolar moments. Amplitude of the moment is in nA.
```

Comment:
Ori is normalized, right? So there should be no unit. And its the dipolar orientation, not moment. Probably we can add somewhere that the dipolar moment in nAm is computed by ori * amplitude[:, None]